tests/e2e/core-tests/specs/merchant/wp-admin-settings-shipping-zones.test.js

```diff
@@ -64,7 +66,7 @@ const runAddNewShippingZoneTest = () => {
 
 			// Add product to cart as a shopper
 			await shopper.goToShop();
-			await shopper.addToCartFromShopPage(simpleProductName);
```

Comment:
Since we don't use the `simpleProductName` variable anymore in this file, we can remove the variable declaration on line 26: https://github.com/woocommerce/woocommerce/pull/30822/files#diff-a2b9ccf9ed6357ae7abd6857d6aca1f2c0ba252731009087d83410d3461be02aL26

Reply:
Thanks for catching all of these. Removed in https://github.com/woocommerce/woocommerce/pull/30822/commits/ef778b898599d50e3775bbdc5909146cb89e4bfa